internal/app/configurable.go

```diff
@@ -378,21 +378,6 @@ func (app *Configurable) Encrypt(parameters map[string]string) interfaces.AppFun
 	}
 
 	switch strings.ToLower(algorithm) {
-	case EncryptAES:
```

Comment:
Also need to remove the `EncryptAES` constant.

Reply:
I have removed the constant, and all its references.